### DISCUSSION TITLE
Add google translate

### DISCUSSION THREAD

**Discussion by SnowCat6** [carson-katri/dream-textures#401] [General]
Add google translate for prompt for all languages. This is simple.
pip install googletrans==4.0.0rc1. 
Just googletrans do errors.

>>> from googletrans import Translator
>>> translator = Translator()
>>> tr=translator.translate('안녕하세요.', dest='en')
>>> tr.text

**Comment by xan2622** [comment]
Maybe this project can help support multi-languages: 
https://github.com/FlagAI-Open/FlagAI/tree/master/examples/AltDiffusion
